README.adoc

```diff
@@ -60,8 +60,8 @@ Google Cloud Platform offers a managed {kube} service called Google Kubernetes E
 needing to install or maintain your own Kubernetes control plane. It provides a hosted {kube} 
 cluster that you can deploy your microservices to. You will use GKE with a Google
 Container Registry (GCR). GCR is a private registry that is used to store 
-and distribute your container images. Note, since GKE is not free, a small
-cost is associated with running this guide. See the official
+and distribute your container images. Note, since GKE is hosted on Google Cloud Platform, a small
+cost may be associated with running this guide. See the official
```

Comment:
Lines 63,64, it’s okay if you don’t want to change the original text since it is consistent with the AWS guide.   If you do change it, I would say "fees may be associated" vs. "a small cost may be associated"

Reply:
Changed to "fees"